controller/api/views.py

```diff
@@ -277,6 +282,17 @@ class ConfigViewSet(ReleasableViewSet):
     model = models.Config
     serializer_class = serializers.ConfigSerializer
 
+    def create(self, request, **kwargs):
+        # Guard against overlapping config changes, using a filesystem lock so that
+        # multiple controller processes can be coordinated.
+        lockfile = os.path.join(tempfile.gettempdir(), kwargs['id'])  # e.g., /tmp/violet-valkyrie
```

Comment:
Good idea. How granular should that be? I'm thinking `/tmp/violet-valkyrie-config` will be good enough, rather than `/tmp/violet-valkyrie-config-create`.